openfisca_country_template/variables/benefits.py

```diff
@@ -75,30 +75,32 @@ class parenting_allowance(Variable):
     value_type = float
     entity = Household
     definition_period = MONTH
-    label = ''' Allowance for low income people with children to care for.
-                Loosely based on the Australian parenting pension'''
-    reference = \
-        [u"https://www.servicesaustralia.gov.au/individuals/services/centrelink/parenting-payment/who-can-get-it"]
+    label = "Allowance for low income people with children to care for."
+    documentation = "Loosely based on the Australian parenting pension."
+    reference = "https://www.servicesaustralia.gov.au/individuals/services/centrelink/parenting-payment/who-can-get-it"
 
     def formula(household, period, parameters):
         '''
         A person's parenting allowance depends on how many dependents they have,
-        How much they, and their partner, earn
+        how much they, and their partner, earn
         if they are single with a child under 8
         or if they are partnered with a child under 6.
         '''
-        family_income = household('household_income', period)
-        income_threshold = 500
-        if family_income > income_threshold:
-            return 0
-        is_single = household.nb_persons(Household.PARENT)
+        parenting_allowance = parameters(period).benefits.parenting_allowance
+
+        household_income = household.sum(household.members('salary', period))
+        income_threshold = parenting_allowance.income_threshold
+        income_condition = household_income < income_threshold
```

Comment:
```suggestion
        income_condition = household_income <= income_threshold
```

Reply:
This looks right to me, and is equivalent to the original, where we exited with 0 allowance if the income was greater than the threshold. This is a bit clearer though, and using the parameter is definitely better than my lazy approach.